src/haddock/gear/clean_steps.py

```diff
@@ -61,12 +62,11 @@ def clean_output(path: FilePath, ncores: int = 1) -> None:
     # Files to delete
     # deletes all except the first one
     files_to_delete = [
-        '.inp',
-        '.inp.gz',
-        '.out',
-        '.out.gz',
-        '.job',
-        '.err',
```

Comment:
I would still leave .err as a file to delete as it can be generated by slurm in batch mode and it makes no sense to keep it.